docs/user/filter_expressions.rst

```diff
@@ -222,6 +222,21 @@ filter expressions.
 
    Boolean specifying whether the user has enabled Do Not Track.
 
+.. js:attribute:: normandy.recipe
+
+   Object containing information about the recipe being checked. Only documented
+   attributes are guaranteed to be available.
+
+   .. js:attribute:: normandy.recipe.id
```

Comment:
Nah I can do that. It makes it mildly more difficult in case we want to use another property of the recipe (I have one future use case of this in mind in particular, simplifying our country/locale/channel filtering), but that's partly the point.